Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1781,20 +1782,23 @@ static HRESULT _CreatePatternBrush(
 
     size_t bitsPerComponent = 8;
     size_t bytesPerRow = 4 * tileSize.size.width;
-    CGBitmapInfo bitmapInfo = kCGImageAlphaPremultipliedFirst | kCGBitmapByteOrder32Big;
+    CGBitmapInfo bitmapInfo = kCGImageAlphaPremultipliedFirst | kCGBitmapByteOrderDefault;
```

Comment:
However, @msft-Jeyaram: If you want to avoid unnecessary bitmap copies, please use AlphaLast|Big: That is `RGBA`, and won't have the overhead of a format-swizzling copy when we give it to D2D.